app/controllers/application_controller.rb

```diff
@@ -5,6 +5,159 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, 'yaaaaaaaaaaaaaaaaaaaaas_queeeeeeeeeeen'
   end
 
-end
\ No newline at end of file
+  # Fwitter Homepage
+  get '/' do
+    erb :index
+  end
+
+  ############# TWEETS #####################
+
+  # READ
+  get '/tweets' do #all tweets
+     if is_logged_in? #if person is logged in...
+      @tweets = Tweet.all # get all the tweets
+      erb :'tweets/tweets'
+    else
+      redirect to '/login' #if not, have them log in
+    end
+  end
+
+  # CREATE
+  get '/tweets/new' do #goes to create page/form
+    if is_logged_in?
+      erb :'tweets/create_tweet' #allow them to create a tweet if they are logged in
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do #takes params from the form
+    if params[:content] == '' #no blank tweets allowed. descriptive error message would be nice
+      redirect to 'tweets/new'
+    else
+      user = User.find_by_id(session[:user_id])
+      @tweet = Tweet.create(content: params[:content], user_id: user.id) #make that tweet
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.